### PR TITLE
Improving documentation

### LINKED ISSUE #473: Documentation: Refactor the getting started section
suggestions:
- Replace "Models" by "Architectures"
- Add section "Models" where we would list all available pre-trained models to download
- move content of "Data" under "Usage"

### LINKED ISSUE #475: Change yes/no by ✅  / ❌
Purely cosmetic: but given that the table [here](https://ivadomed.org/en/latest/purpose.html) is fairly dense, it would help visually to replace the yes/no by colorful emojis.

### PR DESCRIPTION
**Pull request by lrouhier**
This PR aims at improving the documentation as a few section needs to be modified. 

fix #473. 
fix #475  

to visualize: https://ivadomed.org/en/lr-fixing_documentation/

### CODE REVIEW

**Comment by jcohenadad** [comment]
you could use this PR to also fix #475

**Comment by lrouhier** [comment]
I'll do that !

**Comment by lrouhier** [comment]
I guess something like this would work. 
Could we build this doc @jcohenadad ?
image

**Comment by jcohenadad** [comment]
referring to https://github.com/ivadomed/ivadomed/pull/477#issuecomment-709580010:
- could you find a green check not enclosed in a box? i vaguely remember seeing an ivadomed-like JMLR paper doing the same table, with nicer-looking checks/crosses
- could you center the checks/cross
- under uncertainty: could you split the two sub-columns?

**Comment by lrouhier** [comment]
image
 are these the checkmarks you were thinking about ?

**Comment by jcohenadad** [comment]
> are these the checkmarks you were thinking about ?

yup! these are nice. Do you have a red cross that is a bit less aggressive? (same style as the green check)

**Comment by lrouhier** [comment]
like this one ? 
image

**Comment by lrouhier** [comment]
Just to be clear, this is actually HTML Unicode. I created two commands ( |yes| and |no|) that refer to HTML Unicode in purpose.rst. So the table needs to be filled with |yes| and |no| see [here](https://docs.google.com/spreadsheets/d/1_MydnHnlOAuYzJ9QBCvPC9Jq2xUmPWI-XttTfcdtW2Y/edit#gid=0). It is a copy of the original table, because it ia WIP, it will replace the original one after approval.

**Comment by lrouhier** [comment]
Do we have anything else to fix in the doc? otherwise I think it is fairly ready.

**Comment by jcohenadad** [comment]
- @lrouhier pls update wiht master
- could you pls build this branch on RTD and add the link in your PR body so we can all review the rendered doc
thx!

**Comment by lrouhier** [comment]
@jcohenadad I think that you need to trigger the first build (or I need access code)

**Comment by jcohenadad** [comment]
> @jcohenadad I think that you need to trigger the first build (or I need access code)

i just triggered it-- @alexfoias can you add Lucas on the RTD so he can select the build in the future

**Comment by jcohenadad** [comment]
hum, looks like there is a windows/macOS difference in the red cross:

image

**Comment by lrouhier** [comment]
Check now! I forgot to push yesterday evening, I just built it again.

**Comment by jcohenadad** [comment]
neat!
minor fixes:
- ANTSPYNET: Kears --> Keras
- ANTSPYNET: segmentation,clustering --> segmentation, clustering
- column "Data dimension" --> be consistent with space after comma, and replace the "/" with comma

I'll also make suggestions for the "data" section

File: docs/source/models.rst [hunk 1]
```diff
@@ -1,44 +1,59 @@
...
 .. autoclass:: ivadomed.models.UNet3D
 
 
 :mod:`Countception`
----------------------------
+^^^^^^^^^^^^^^^^^^^
 
 .. autoclass:: ivadomed.models.Countception
 
+Models
```
Comment (jcohenadad) [inline docs/source/models.rst:48]: it's a bit "hidden"-- i suggest putting it as main section, and renaming it "Pre-trained models"

File: docs/source/models.rst [hunk 2]
```diff
@@ -1,44 +1,59 @@
-Models
-======
+Architectures
```
Comment (jcohenadad) [inline docs/source/models.rst:1]: the file should be renamed architectures.rst

**Comment by jcohenadad** [comment]
links to example config files are broken-- i'm fixing it

File: docs/source/models.rst (hunk 2 above)
Comment (lrouhier) [reply docs/source/models.rst:3]: done in 
9eb7885

**Comment by jcohenadad** [comment]
@lrouhier about https://github.com/ivadomed/ivadomed/pull/477/commits/9eb78850a153ffd014dbe6d6ad2ad4ddccc49901: i've noticed this renaming broke the hard-coded path to models.rst in ivadomed's code.

**Comment by lrouhier** [comment]
It broke something in the code? I know it broke something in the pupose.rst (which I changed) 
I have just changed what was needed in index.rst.

**Comment by jcohenadad** [comment]
> It broke something in the code? I know it broke something in the pupose.rst (which I changed)

my bad, looking good 👍

**Comment by jcohenadad** [comment]
purpose table: niftytorch: Classificication --> Classification

**Comment by jcohenadad** [comment]
purpose table: DL base library
what is the difference between "Keras (backend TF)" and "Tensorflow/Keras"?

**Comment by jcohenadad** [comment]
purpose: table: https://github.com/neuropoly/ivado-medical-imaging --> https://github.com/ivadomed/ivadomed

**Comment by lrouhier** [comment]
> purpose table: DL base library
> what is the difference between "Keras (backend TF)" and "Tensorflow/Keras"?

I don't think there is one. I think it just depends on who filled the row.

**Comment by jcohenadad** [comment]
> > purpose table: DL base library
> > what is the difference between "Keras (backend TF)" and "Tensorflow/Keras"?
> 
> I don't think there is one. I think it just depends on who filled the row.

so it should be changed to be consistent across cells

**Comment by lrouhier** [comment]
I modified the table so it is consistent. I will check for typo and/or similar cases after compilation.

**Review by jcohenadad — APPROVED** [review]

**Pull request merged**